grpc/client.go

```diff
@@ -58,15 +58,10 @@ func ClientSetup(c *cmd.GRPCClientConfig, tlsConfig *tls.Config, statsRegistry p
 		return nil, err
 	}
 
-	var balancerName = roundrobin.Name
-	if c.LoadBalancerOverride != "" {
-		balancerName = c.LoadBalancerOverride
-	}
-
 	creds := bcreds.NewClientCredentials(tlsConfig.RootCAs, tlsConfig.Certificates, hostOverride)
 	return grpc.Dial(
 		target,
-		grpc.WithDefaultServiceConfig(fmt.Sprintf(`{"loadBalancingConfig": [{"%s":{}}]}`, balancerName)),
+		grpc.WithDefaultServiceConfig(fmt.Sprintf(`{"loadBalancingConfig": [{"%s":{}}]}`, roundrobin.Name)),
```

Comment:
Do we still need WithDefaultServiceConfig? We expect to always set it in our resolver, right?

Reply:
Not quite. The default `dns` resolver as well as our own `static` and `srv` resolvers do not return an `loadBalancingConfig` when calling `cc.UpdateState()`.